src/webgl/p5.RendererGL.js

```diff
@@ -2113,7 +2281,7 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
       // this.activeImageLight has image as a key
       // look up the texture from the diffusedTexture map
       let diffusedLight = this.getDiffusedTexture(this.activeImageLight);
-      shader.setUniform('environmentMapDiffused', diffusedLight);
+      shader.setUniform('environmentMapDiffusedCubemap', diffusedLight);
```

Comment:
I am not sure but I feel diffusedLight is also a sampler2D texture, could it be passed as a uniform with samplerCube texture?

Reply:
Here, diffusedLight is obtained from `getDiffusedTexture`, which returns a `CubemapTexture` object. Since `environmentMapDiffusedCubemap` in the shader expects a cubemap texture (samplerCube), so I think  `diffusedLight` is being correctly assigned to the `environmentMapDiffusedCubemap` uniform in the shader, which is of type `samplerCube`.

Reply:
Oooh...sorry. I overlooked this, previously we had diffusedLight with `sampler2D` texture so I got confused. I see you have updated getDiffusedTexture()